aldryn_gallery/models.py

```diff
@@ -27,6 +27,9 @@ class GalleryPlugin(CMSPlugin):
 
     style = models.CharField(
         _('Style'), choices=STYLE_CHOICES + get_additional_styles(), default=STANDARD, max_length=50)
+    extra_styles = models.CharField(
+        _('Extra styles'), max_length=50,
+        help_text=_('An arbitrary string of CSS classes to add'))
```

Comment:
is this field is required?
if no - please add ``blank=True`` or a default value